cli-plugins/apps/how-to-guides.hbs.md

```diff
@@ -0,0 +1,611 @@
+    tanzu-java-w… │
+    tanzu-java-w… │      Step 1 - 8.87s (Deploying)
+    tanzu-java-w… │      DONE IN: 8.87s
+    tanzu-java-w… │
+    tanzu-java-w… │
+    tanzu-java-w… │ Tracking new pod rollout (tanzu-java-web-app-build-1-build-pod):
+    tanzu-java-w… │      ┊ Scheduled       - (…) Pending
+    tanzu-java-w… │      ┊ Initialized     - (…) Pending
+    tanzu-java-w… │      ┊ Ready           - (…) Pending
+    ...
+    ```
+
+<!-- add info regarding debug flag -->
+
+## <a id='export-usage'> --export usage
+
+When using `--export` flag, the user can retrieve the workload pared out of some fields such as status and metadata fileds as `creationTimestamp`.
+This means that the workload is retrieved with the fields that it was specifically applied with. 
+
+For example, if user creates a workload with
+
+```bash
+tanzu apps workload apply rmq-sample-app --git-repo https://github.com/jhvhs/rabbitmq-sample --git-branch main --service-ref "rmq=rabbitmq.com/v1beta1:RabbitmqCluster:example-rabbitmq-cluster-1" -t web
+```
+
+When querying the workload with `--export` flag (defaulting export format is yaml), the following output is what will be displayed:
+
+```bash
+# with yaml format
+    tanzu apps workload get rmq-sample-app --export
+    ---
+    apiVersion: carto.run/v1alpha1
+    kind: Workload
+    metadata:
+    labels:
+        apps.tanzu.vmware.com/workload-type: web
+    name: rmq-sample-app
+    namespace: default
+    spec:
+    serviceClaims:
+    - name: rmq
+        ref:
+        apiVersion: rabbitmq.com/v1beta1
+        kind: RabbitmqCluster
+        name: example-rabbitmq-cluster-1
+    source:
+        git:
+        ref:
+            branch: main
+        url: https://github.com/jhvhs/rabbitmq-sample
+
+# with json format
+    tanzu apps workload get rmq-sample-app --export --output json
+    {
+        "apiVersion": "carto.run/v1alpha1",
+        "kind": "Workload",
+        "metadata": {
+            "labels": {
+                "apps.tanzu.vmware.com/workload-type": "web"
+            },
+            "name": "rmq-sample-app",
+            "namespace": "default"
+        },
+        "spec": {
+            "serviceClaims": [
+                {
+                    "name": "rmq",
+                    "ref": {
+                        "apiVersion": "rabbitmq.com/v1beta1",
+                        "kind": "RabbitmqCluster",
+                        "name": "example-rabbitmq-cluster-1"
+                    }
+                }
+            ],
+            "source": {
+                "git": {
+                    "ref": {
+                        "branch": "main"
+                    },
+                    "url": "https://github.com/jhvhs/rabbitmq-sample"
+                }
+            }
+        }
+    }
+```
+
+On the other hand, if user wants to retrieve the workload as its currently in the cluster, with its status and all its fields, the way is to get it using `--output` flag. 
```

Comment:
If it's desired to retrieve the workload including all the cluster-specifics (with its status and all its fields) the `--output` flag should be provided.